policyengine_us/variables/gov/states/la/tax/income/exemptions/la_aged_exemption.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class la_aged_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana aged exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.legis.la.gov/legis/Law.aspx?d=102133"
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.la.tax.income.exemptions.aged
+        aged_head = (tax_unit("age_head", period) >= p.thresholds[-1]).astype(
```

Comment:
this exemption is capped at the retirement income - not just a flat amount. The filers can not have a exemption amount higher than their retirement income